### ISSUE TITLE
Impossible to download helper python script

### ISSUE THREAD

**Issue by AndydeCleyre** [darktrojan/openwith#161]
OpenWith `7.0.2`
Firefox `56.0`
Arch Linux

1. Go to OpenWith preferences
2. Click on "Click here to download" (does nothing)
3. Right click and save as . . . and choose a destination (does nothing)

**Comment by darktrojan** [comment]
There's something weird that happens with that but I don't know what it is. I've never had the right-click method fail though.

You could try dragging the link to the address bar, or failing that, just get the file from this repo. It's at `webextension/native/open_with_linux.py`.

**Comment by AndydeCleyre** [comment]
Thank you, dragging to the address bar worked perfectly. Whether you choose to close this issue or keep it open for now, I'm happy.

**Comment by rmlrml** [comment]
FWIW same issue here on

OpenWith `7.0.2`
Firefox `56.0`
Debian GNU/Linux `4.13.10-1`

and the drag-to-address-bar workaround worked for me as well.

Also I don't know if I should have opened a separate issue for this, but I thought I'd mention that it might be worth including the initial "./" in the code portion of step 3 of your installation instructions, i.e.:

> Run the file with the argument "install", like this:
> 
> `./open_with_linux.py install`

At least, that's how I had to run it.

**Comment by darktrojan** [comment]
@rmlrml Yeah, I probably should add the `./`, since it is kinda necessary. :-/
